### ISSUE TITLE
minor issue: UltiSnips requires Vim >= 7.4

### ISSUE THREAD

**Issue by yilongli** [JBakamovic/yavide#29]
I didn't notice this until later in my installation since all requirement checks were passed.

**Comment by JBakamovic** [comment]
Yeah, I will need to address these kind of issues in future. I was already thinking about tracking plugin versions and being able to revert to last good configuration if something breaks.
